lib/execution_engine2/utils/SDKMethodRunner.py

```diff
@@ -608,13 +618,16 @@ def get_job_status(self, job_id, ctx):
 
         return returnVal
 
-    def finish_job(self, job_id, ctx, error_message=None, job_output=None):
+    def finish_job(
+        self, job_id, ctx, error_message=None, job_output=None, finish_code=None
```

Comment:
also I didn't see `finish_code` is used anywhere

Reply:
Not yet, but in the next PR